card/Gemfile

```diff
@@ -0,0 +1,6 @@
+source 'http://rubygems.org'
+
+gemspec
+
+#gem 'card', :path=>'.'
```

Comment:
in general our approach with rake is increasingly that rake tasks should be performed from a dev deck